collectors/statsd.plugin/statsd.c

```diff
@@ -699,9 +699,9 @@ static inline void statsd_process_dictionary(STATSD_METRIC *m, const char *value
                 m->dictionary.base = t;
                 m->dictionary.unique++;
             }
+            t->count++;
```

Comment:
This is a false alarm. The dictionary options are set so that the if an item is created `tmp` is copied into the dictionary. So, it is ok for `tmp` to be allocated in the stack.

The whole point of dictionaries use in this code is to let the dictionary handle all the allocations, so that by deleting the dictionary everything is fred.

And yes, the counter should not be moved.

Just silence this at coverity.